releases/release-1.21/release-notes/maps/pr-97543-map.yaml

```diff
@@ -1,3 +1,3 @@
 pr: 97543
 releasenote:
-  action_required: true
+  text: remove LegacyNodeRoleBehavior and mv ServiceNodeExclusion to GA
```

Comment:
I can see the intent but I'm worried that this doesn't represent the actual change.

Reply:
to elaborate -- the wording "unconditionally enabled" is important for feature gates, roughly:

- alpha: default to false, can be enabled if consumers choose to
- beta: default to true, can be disabled if consumers choose to
- generally available (GA): always true, cannot be turned off